### PR TITLE
docs: watch does not work in pipe

### PR DESCRIPTION
**Pull request by gossion**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

/kind documentation
> /kind test
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Fix a tiny doc issue: `watch` does not work in pipe

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```

```

### CODE REVIEW

File: docs/install-csi-driver-master.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi
 - check pods status:
 
 ```
-watch kubectl get po -o wide -n kube-system | grep csi-azuredisk
```
Comment (bcho) [inline docs/install-csi-driver-master.md]: How about use `kubectl get -w` ?

```
$ kubectl get po -o wide -n kube-system -l 'app=csi-azuredisk-controller'
```

**Comment by andyzhangx** [comment]
@chewong  it's running out of quota, could you clean up the testing subs? thanks.
And in long term, how can I clean up to release quota by myself, could you provide guide? thanks.

```
error creating cluster: cannot deploy: cannot create deployment: resources.DeploymentsClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="InvalidTemplateDeployment" Message="The template deployment 'kubetest-bd3eb62b-285c-11ea-aec4-9eddc3436cf9' is not valid according to the validation procedure. The tracking id is '1c4efd86-07d2-4dd3-b308-53e1a722d91a'. See inner errors for details." Details=[{"code":"QuotaExceeded","message":"The operation couldn't be completed as it results in exceeding quota limit of standardDSv2Family Cores. Maximum allowed: 200, Current in use: 198, Additional requested: 6. Read more about quota limits at https://aka.ms/AzurePerVMQuotaLimits. Submit a request for Quota increase using the link https://aka.ms/ProdportalCRP/?#create/Microsoft.Support/Parameters/%7B%22subId%22:%220e46bd28-a80f-4d3a-8200-d9eb8d80cb2e%22,%22pesId%22:%2206bfd9d3-516b-d5c6-5802-169c800dec89%22,%22supportTopicId%22:%22e12e3d1d-7fa0-af33-c6d0-3c50df9658a3%22%7D."}]
```

File: docs/install-csi-driver-master.md (hunk 1 above)
Comment (feiskyer) [reply docs/install-csi-driver-master.md]: +1 of get --watch, e.g.

```
kubectl -n kube-system get pod -o wide --watch -l app=csi-azuredisk-controller
kubectl -n kube-system get pod -o wide --watch -l app=csi-azuredisk-node
```

**Comment by chewong** [comment]
> @chewong it's running out of quota, could you clean up the testing subs? thanks.
> And in long term, how can I clean up to release quota by myself, could you provide guide? thanks.
> 
> ```
> error creating cluster: cannot deploy: cannot create deployment: resources.DeploymentsClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="InvalidTemplateDeployment" Message="The template deployment 'kubetest-bd3eb62b-285c-11ea-aec4-9eddc3436cf9' is not valid according to the validation procedure. The tracking id is '1c4efd86-07d2-4dd3-b308-53e1a722d91a'. See inner errors for details." Details=[{"code":"QuotaExceeded","message":"The operation couldn't be completed as it results in exceeding quota limit of standardDSv2Family Cores. Maximum allowed: 200, Current in use: 198, Additional requested: 6. Read more about quota limits at https://aka.ms/AzurePerVMQuotaLimits. Submit a request for Quota increase using the link https://aka.ms/ProdportalCRP/?#create/Microsoft.Support/Parameters/%7B%22subId%22:%220e46bd28-a80f-4d3a-8200-d9eb8d80cb2e%22,%22pesId%22:%2206bfd9d3-516b-d5c6-5802-169c800dec89%22,%22supportTopicId%22:%22e12e3d1d-7fa0-af33-c6d0-3c50df9658a3%22%7D."}]
> ```

Looks like #227 has caused a lot of resource groups not being cleaned up properly since the author kept forcing pushing commits before the e2e test is finished. The cleanup tool only cleans up resource groups that are older than 3 days old so that's why a lot of them haven't been cleaned up yet (The PR was merged 2 days ago). I've manually cleaned up unused resource groups and will write docs on how to do that.

**Comment by feiskyer** [comment]
@andyzhangx this are probably existing code issues?

**Comment by andyzhangx** [comment]
> @andyzhangx this are probably existing code issues?

looks like this an issue in `hack/verify-golint.sh`, it suddenly does not work today, it is always working before today, I will disable it first.

```
# hack/verify-golint.sh
WARN [runner] Can't run linter unused: buildssa: analysis skipped: errors in package: [/root/go/src/sigs.k8s.io/azuredisk-csi-driver/test/e2e/pre_provisioning.go:59:14: undeclared name: azurediskDriver /root/go/src/sigs.k8s.io/azuredisk-csi-driver/test/e2e/pre_provisioning.go:74:16: undeclared name: azurediskDriver /root/go/src/sigs.k8s.io/azuredisk-csi-driver/test/e2e/pre_provisioning.go:113:16: undeclared name: azurediskDriver /root/go/src/sigs.k8s.io/azuredisk-csi-driver/test/e2e/dynamic_provisioning.go:273:15: undeclared name: azurediskDriver]
ERRO Running error: buildssa: analysis skipped: errors in package: [/root/go/src/sigs.k8s.io/azuredisk-csi-driver/test/e2e/pre_provisioning.go:59:14: undeclared name: azurediskDriver /root/go/src/sigs.k8s.io/azuredisk-csi-driver/test/e2e/pre_provisioning.go:74:16: undeclared name: azurediskDriver /root/go/src/sigs.k8s.io/azuredisk-csi-driver/test/e2e/pre_provisioning.go:113:16: undeclared name: azurediskDriver /root/go/src/sigs.k8s.io/azuredisk-csi-driver/test/e2e/dynamic_provisioning.go:273:15: undeclared name: azurediskDriver]
```

**Pull request merged**